Kernel/TeukolskyMode.m

```diff
@@ -126,8 +114,8 @@
   assoc = <| "s" -> s, 
 		     "l" -> l,
 		     "m" -> m,
-		     "n"->n,
-		     "k"->k,
+		     "n" -> n,
```

Comment:
This doesn't seem like a relevant change?

Reply:
I think this was just a neatness thing for lines of code to line up. It shouldn't change anything significantly?